### ISSUE TITLE
Inconsistent Results 

### ISSUE THREAD

**Issue by ibrahimsadek** [google/rekall#430]
We have several Windows memory dump files (Extension AFF4) . When we used Rekall Version 1.6.0 (Gotthard) to extract the physical addresses of some processes (i.e., rekall memmap --proc_regex "$payloadName" -f "$entry" &> "$textfile"), several physical addresses were out of range for some memory files. 

Recently, we employed Rekall Version 1.7.3.dev33 (Hurricane Ridge) to the same memory files. However, we obtained different physicals addresses than the formal Rekall and they seem to be within the range of the memory files.

Can you please explain, why this inconsistency might happen ? should we ignore the results obtained with the formal Rekall and we better use the recent Rekall ?

Thank you.

**Comment by scudette** [comment]
We do fix bugs all the time. Without knowing the specifics it is possible that recent versions fixed a bug you don't see with the older version. It sounds like you are saying that the new version produces more reasonable result?

I vaguely remember some bug like that in the address translation code?

**Comment by ibrahimsadek** [comment]
Yes, it seems that the new version produce more reasonable results.

[snapshot-2017-08-02_13-47-0.aff4_old.txt](https://github.com/google/rekall/files/2255846/snapshot-2017-08-02_13-47-0.aff4_old.txt)

[snapshot-2017-08-02_13-47-0.aff4_new.txt](https://github.com/google/rekall/files/2255849/snapshot-2017-08-02_13-47-0.aff4_new.txt)


I attached results for the same memory file.
